sentry-spring/src/main/java/io/sentry/spring/tracing/SentrySpanAdvice.java

```diff
@@ -26,6 +26,7 @@ public SentrySpanAdvice(final @NotNull IHub hub) {
     this.hub = Objects.requireNonNull(hub, "hub is required");
   }
 
+  @SuppressWarnings("deprecation")
```

Comment:
Accidentally added?

Reply:
no, See https://github.com/getsentry/sentry-java/pull/1320/files#r597775167 (and other comments maybe too for context)